### DISCUSSION TITLE
IP/app call over NGINX does not work

### DISCUSSION THREAD

**Discussion by Lacsap93** [flet-dev/flet#1213] [Q&A]
Hello all
Here is what I would like to achieve: I want to control a flet program running on my Raspberry 3B+ from meim cell phone. For this I have installed NGINX on the Raspberry. Now when I type localhost in the internet explorer on the Raspberry, the text "Welcome to NGINX...." appears. ". And when I type localhost/app, the program Counter.py appears, which is available on the flet website. 

Unfortunately it doesn't work with the mobile yet. I made a hotspot with my mobile and connected the RPI to it.

Then I entered the IP of my RPI on my cell phone. Then also the text "Welcome to NGINX...." appears. That would be good. But if I add an /app after the IP then only the pulsating text "Loading..." appears. Nothing else happens. I tested this with Firefox and Chrome on a cell phone and on a PC. The result is always the same.

Can someone help me what the problem is? 

Since my English is not very good, I wrote this with the help of a translator. I hope that it is somewhat understandable.

**Comment by Lacsap93** [comment]
can someone help me with my problem?

**Reply by rkleivel** [reply]
Hi! I might be able to help you. Could you please post the nginx config and at least how you start your app with flutter? (ft.app(name... , port... , view... ))
